doc/user/content/sql/deallocate.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "DEALLOCATE"
+description: "`DEALLOCATE` clears a prepared statement."
+menu:
+  main:
+    parent: "sql"
+---
+
+`DEALLOCATE` clears [prepared statements](../prepare) that have been created during the current session. You can use this to reclaim memory, but even without an explicit `DEALLOCATE` command, all prepared statements will be cleared at the end of a session.
```

Comment:
I'd remove the sentence about reclaiming memory, because it's reclaiming a few kilobytes, nothing that matters.